cms/djangoapps/contentstore/views/tests/test_bulksettings.py

```diff
@@ -211,19 +206,21 @@ def do_values_match(self, setting_types, child, computed_settings):
         """
 
         for setting_type in setting_types:
-            setting= getattr(child, setting_type)
+            setting = getattr(child, setting_type)
 
             if setting_type == 'start' or setting_type == 'due':
                 setting = setting.strftime('%m/%d/%Y')
 
-            self.assertEqual(setting, computed_settings[setting_type],
-                            msg="setting_type" + str(setting_type) + "does not match")
+                self.assertEqual(
+                    setting,
+                    computed_settings[setting_type],
+                    msg="setting_type" + str(setting_type) + "does not match"
```

Comment:
I know you didn't write this, but let's clean it up while we're touching this line anyway..

This should use string formatting instead of concatenation.

The benefits are:
- often faster
- easier to read
- more flexible

It also helps you catch, well, formatting issues, like in this example. Here, the concatenated strings _should_ have had white space added. Instead, this error will mistakenly write `setting_typeSETTING_TYPEdoesnot match` instead of `setting_type SETTING_TYPE does not match`. See the difference? 


```python
msg="setting_type {setting_type} does not match".format(
    setting_type=setting_type,
),
```